src/components/Settings/RecorderSettings.tsx

```diff
@@ -32,6 +33,19 @@ export const RecorderSettings = () => {
     clearErrors('recorder.browserPath')
   }
 
+  const isValidPath = (path?: string) => {
+    if (!path) return false
+    const validPaths = [
+      'chrome.app',
+      'chromium.app',
+      'chrome.exe',
+      'chromium.exe',
+      '/chrome',
+      '/chromium',
+    ]
```

Comment:
We should probably include chromium derived browsers, for example Edge 🤔 
I wonder if there's a better way to detect a working browser, maybe like doing a test run in the background and checking that a request is actually recorded, might be too laborious thou 😄

Reply:
> We should probably include chromium derived browsers, for example Edge 🤔

I wouldn't do it at this stage. If the user wants to try another Chromium-based browser, they should do it at their own risk.
As far as I understand, we don't prevent the user from selecting a different browser, so they can still try to use other browsers.

Reply:
We haven't really tested in other Chrome-based browsers and potential ramifications they introduce so I'd say it's safer to just go with Chrome and Chromium for now.